monitoring/terraform/modules/slack_chatbot/iam.tf

```diff
@@ -54,7 +52,10 @@ data "aws_iam_policy_document" "sns_topic_policy" {
     condition {
       test     = "StringEquals"
       variable = "aws:SourceAccount"
-      values   = [data.aws_caller_identity.current.account_id]
+      values = [
```

Comment:
Instead of hard-coding these values, can you get them from provider context?

```hcl
data "aws_caller_identity" "platform" {
    provider = aws.experience
}
data "aws_caller_identity" "experience" {
  provider = aws.experience
}
```

Use like:

```hcl
values   = [data.aws_caller_identity.platform.account_id, data.aws_caller_identity.experience.account_id]
```